src/utils/fixtures-resolver.util.js

```diff
@@ -0,0 +1,62 @@
+import EnvironmentUtil from './environment.util';
+import { CustomerFixture } from '../fixtures/customer.fixture';
+import { CustomerStaticFixture } from '../static-fixtures/customer.static-fixture';
+import { ProductStaticFixture } from '../static-fixtures/product.static-fixture';
+import { FullProductFixture } from '../fixtures/full-product.fixture';
+import { CartFixture } from '../fixtures/cart.fixture';
+import { CartStaticFixture } from '../static-fixtures/cart.static-fixture';
+import { CmsPageFixture } from '../fixtures/cms-page.fixture';
+import { CmsPageStaticFixture } from '../static-fixtures/cms-page.static-fixture';
+
+export default class FixturesResolver {
```

Comment:
In general it's wrong when utils aware of real implementations, move it to parent fixture class and modify tests accordingly